pkg/devfile/adapters/common/command.go

```diff
@@ -206,7 +253,13 @@ func ValidateAndGetPushDevfileCommands(data data.DevfileData, devfileInitCmd, de
 	} else if !isInitCmdEmpty && initCmdErr == nil {
 		isInitCommandValid = true
 		commandMap[common.InitCommandGroupType] = initCommand
-		klog.V(4).Infof("Init command: %v", initCommand.Exec.Id)
+		var commandID string
+		if initCommand.Composite != nil {
```

Comment:
you can use your helper func `GetID()`?

Reply:
What's funny is that these code blocks is what made me implement `GetID` as a helper function, but then I forgot to update these 😆

Reply:
Done